com.unity.render-pipelines.high-definition/Editor/VFXGraph/VFXHDRPBinder.cs

```diff
@@ -14,14 +20,156 @@ class VFXHDRPBinder : VFXSRPBinder
         public override string SRPAssetTypeStr  { get { return typeof(HDRenderPipelineAsset).Name; } }
         public override Type SRPOutputDataType  { get { return typeof(VFXHDRPSubOutput); } }
 
-        public override void SetupMaterial(Material mat)
+        static Dictionary<HDShaderUtils.ShaderID, string> s_ShaderNames = new Dictionary<HDShaderUtils.ShaderID, string>()
+        {
+            { HDShaderUtils.ShaderID.SG_Lit,      "Lit"      },
```

Comment:
Poking @slunity for this change as well

Reply:
Thanks for the heads up. Well the new HDSubTargets in my changes can directly provide the resetter function so even better.